### ISSUE TITLE
Suggestion make a demo site showing what the panel looks like 

### ISSUE THREAD

**Issue by DARKBEAM1977** [real2two/dashactyl#3]
Please make a demo site showing what the panel looks like from admi perspective and a iser perspective

**Comment by real2two** [comment]
I currently don't have enough funds to purchase a VPS to do all that but I might do it when it is possible.

**Comment by DARKBEAM1977** [comment]
Hey i think this migjt look like advertising but im support in a free web hosting company maybe you can consider that? Btw im luffy#0002

**Comment by real2two** [comment]
I might simulate a fake dashboard that looks like Dashactyl to make this.

**Comment by DARKBEAM1977** [comment]
Yeah thats what im requesting it musnt be a actual hosting service

**Comment by DARKBEAM1977** [comment]
But it should just show the features of the client panel

**Comment by real2two** [comment]
I'll be trying to make this after v1.

**Comment by Tooxic** [comment]
> I currently don't have enough funds to purchase a VPS to do all that but I might do it when it is possible. (someone lets me borrow a Pterodactyl Panel to test currently)

I could possibly give you a small VPS for free.

**Comment by dudeisbrendan03** [comment]
Don't bother, take screenshots or generate a static page and throw it onto something like Netlify.
